indico/queries/teach_tasks.py

```diff
@@ -0,0 +1,244 @@
+import json
+from typing import List, Any
+
+import pandas as pd
+
+from indico.queries.datasets import CreateDataset, GetDataset
+
+from indico.client.request import (
+    GraphQLRequest,
+    RequestChain,
+)
+
+from indico.types.questionnaire import Questionnaire, Example
+from indico.types.dataset import Dataset
+from indico.errors import IndicoNotFound, IndicoInputError
+
+
+class AddLabels(GraphQLRequest):
+    """
+    Add labels to an existing labelset.
+
+    Args:
+        dataset_id (int): The id of the dataset to add labels to.
+        labelset_id (int): The id of the labelset to add labels to.
+        target (List[Any]): A list of labels to add to the labelset.
+        row_index (List[Any]): The row indices corresponding with each item in targets.
```

Comment:
The interface here could require labels in the dictionary format. What made us choose the list version over just asking for the dictionary? I'm wondering if this interface might be more error prone and allow for mis ordered labels but was it more convenient in some contexts?